vizro-core/changelog.d/20230904_174834_nadija_ratkusic_graca.md

```diff
@@ -0,0 +1,42 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
```

Comment:
Do we need to update this file as well? @maxschulz-COL

Reply:
I thought about this shortly and I think the answer is no.

Until we do our first release, all the "changes" are not really changes, as they essentially alter the base-line.

Once we have released I think this would be something that should be added to the CHANGELOG, as we are alterning the appearance, but for now no